transformations/question_neural_paraphaser/README.md

```diff
@@ -0,0 +1,38 @@
+# Dialog Rephrasing transformation
+This perturbation uses a model to rephrase a sentence in a natural manner and was proposed for paraphrasing questions.
+
+
+Author name: Pierre Colombo and Emile Chapuis
+Author email: colombo.pierre@gmail.com and emile.chapuis@gmail.com
+Author Affiliation: Telecom Paris
+
+## What type of a transformation is this?
+This transformation uses a neural model finetuned for question rephrasing. 
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification, 
+text generation, etc. It has been proposed for question paraphrasing.
+
+## Previous Work
+This work relies on T5 and was finetuned following ``https://towardsdatascience.com/paraphrase-any-question-with-t5-text-to-text-transfer-transformer-pretrained-model-and-cbb9e35f1555``
+```bibtex
```

Comment:
Another previous PR uses the exact model but in a Question Answering context. You might want to mention about that in the related work.
https://github.com/GEM-benchmark/NL-Augmenter/tree/main/transformations/quora_trained_t5_for_qa